tremor-pipeline/src/lib.rs

```diff
@@ -90,12 +133,19 @@ pub struct MetricsChannel {
     tx: Sender<MetricsMsg>,
     rx: Receiver<MetricsMsg>,
 }
+/// Channel for plugging-logging messages
+pub struct LoggingChannel {
```

Comment:
Since `LoggingChannel` and `MetricsChannel` are effectively the same it would be nicer to implement them as:

```rust
struct OverflowingChannel<Msg> {
    tx: Sender<Msg>,
    rx: Receiver<Msg>
}
```

instead of having two identical copies of code

Reply:
This too is updated, nice thank you  again :)